util/endpointParams.js

```diff
@@ -0,0 +1,34 @@
+const { chainTableNames } = require('./constants.js');
+const { dbQueryAvgTxCost } = require('./activity-report/index.js');
+
+function verifyChain(chain) {
+  if (!chain || !chainTableNames[chain]) {
+    throw new Error(`chain '${chain}' is not supported`);
+  }
+}
+
+module.exports = {
+  'api/v1/rsk-activity-report/avg-tx-cost': {
```

Comment:
:+1:

Reply:
so this is both an API route and a cache key